### PR TITLE
Add story image to Today in Focus podcast for emails

### PR DESCRIPTION
**Pull request by JustinPinner**
## What does this change?
In 'normal' dotcom frontend, for the Today in Focus podcast we load an image related to the podcast instead of using the default 'album art' image. 

This change has also been made to the email front, too.

## Screenshots
**Before:**
screen shot 2019-02-21 at 16 11 09

**After (Switched OFF):**
screen shot 2019-02-25 at 12 14 07

**After (Switched ON):**
screen shot 2019-02-25 at 12 15 39

## What is the value of this and can you measure success?
We maintain consistency between the web front and an emailed view of that. We _may_ see an uplift in engagement from emails because of this more appropriate imagery.

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [x] Other (please specify) **EMAILS**

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

cc @tomrf1 The test I've added to discover if the podcast is present is failing. That may be because the pressed page doesn't have the podcast in it, or there's more code that I need to update before it'll work. Facia is confusing, after all. I've opened this as a draft PR while we figure this out. Any pointers you can give would be appreciated! Thanks.

### CODE REVIEW

**Comment by ajwl** [comment]
This all *seems* very reasonable. How can you test emails?

File: facia/test/FaciaControllerTest.scala [hunk 1]
```diff
@@ -224,4 +225,23 @@ import scala.concurrent.Await
     responseHeaders("Surrogate-Control") should include("max-age=900")
   }
 
+  // TODO - devise a test that works!
+  it should "render email fronts including the TiF podcast with story-related image and not the default album art" in {
+    conf.switches.Switches.FlagshipEmailContainerSwitch.switchOn()
```
Comment (regiskuckaertz) [inline facia/test/FaciaControllerTest.scala]: Should these switches be restored to their previous state at the end of the test?

**Review by regiskuckaertz — APPROVED** [review]
One minor question but not blocking, let's 🚀

File: facia/test/FaciaControllerTest.scala (hunk 1 above)
Comment (JustinPinner) [reply facia/test/FaciaControllerTest.scala]: I've removed this test so no longer a problem :)

**Pull request merged**